exercises/tournament/description.md

```diff
@@ -60,8 +60,3 @@ Devastating Donkeys;Courageous Californians;draw
 ```
 
 Means that the Devastating Donkeys and Courageous Californians tied.
-
-Your program should only process input lines that follow this format.
```

Comment:
Thanks @petertseng , good points.

> "should I expect invalid lines?"

Isn't "invalid input will not be tested" up to the tracks to decide, since they are free to add or remove tests. 

>  Wouldn't it be the case that forcing the student to look at every test case to answer that question is going against the one-test-at-a-time metholodogy?

Why would they be writing code to handle it without a failing test first? Once they get to the end they might notice that they have not had to handle invalid input.

(I'm not against adding something about test input validity to the description.)